### PR TITLE
Filling gaps in the "return value" section.

### PR DESCRIPTION
**Pull request by lslachciak**
[Problem] Some methods did not have a "return section" in their description.
[Solution] Add missing sections.

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/596

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Drawer.htm [hunk 1]
```diff
@@ -254,8 +254,19 @@ <h3>Summary</h3>
 					<div class="returntype">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Drawer.htm]: It looks like "No Return Value" is written for the case of no return.

However only in this page "No return value" is used in line 209 and line 326.
How about changing "return" to "Return" in this page?

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm [hunk 2]
```diff
@@ -260,6 +260,14 @@ <h3>Summary</h3>
                 </table>
             </div>
 
+            <div class="returntype">
+                <p><span class="return">Return value:</span></p>
+
+
+                No Return Value
```
Comment (TomaszLukawskiSam) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm]: Return Value -> return value

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm [hunk 3]
```diff
@@ -302,6 +310,13 @@ <h3>Summary</h3>
                 <pre class="signature prettyprint">pop() </pre>
             </div>
 
+            <div class="returntype">
+                <p><span class="return">Return value:</span></p>
+
+
+                No Return Value
```
Comment (TomaszLukawskiSam) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm]: Return Value -> return value

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Progress.htm [hunk 4]
```diff
@@ -305,6 +322,13 @@ <h3>Summary</h3>
 				<p>
 				</p>
 			</div>
+			<div class="returntype">
+				<p><span class="return">Return value:</span></p>
+
+
+				No Return Value
```
Comment (TomaszLukawskiSam) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Progress.htm]: Return Value -> return value

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm (hunk 2 above)
Comment (mijin85cho) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm]: @TomaszLukawskiSam Hello, I have a question. I can see in other pages, "Return value:" is used. Are you going to use "return value" in other pages, too?

Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm]: > @TomaszLukawskiSam Hello, I have a question. I can see in other pages, "Return value:" is used. Are you going to use "return value" in other pages, too?

We will do it to make it uniform throughout the whole project, but at the moment we have too much pending PR, so first we have to fix and merge current PRs.

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm (hunk 3 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm]: Done

**Pull request merged**